scripts/trim_tract_cat.py

```diff
@@ -55,11 +56,20 @@ def make_trim_file(infile, output_file=None, output_dir=None,
     columns_to_keep = DummyDC2ObjectCatalog(schema_version).required_native_quantities
 
     patches = []
-    with pd.HDFStore(infile, 'r') as fh:
+    if verbose:
+        print("Reading: ", infile)
+    with pd.HDFStore(infile, 'r', errors='replace') as fh:
         for key in fh:
+            if verbose:
+                print("Key: ", key)
             if not re.match(GROUP_PATTERN, key.lstrip('/')):
                 continue
-            load_trim_save_patch(output_file, fh, key, columns_to_keep)
+            try:
+                load_trim_save_patch(output_file, fh, key, columns_to_keep)
+            except UnicodeDecodeError as e:
```

Comment:
Is `UnicodeDecodeError` actually expected and should be skipped, or is this just a hack for now and we should come up with a better solution later? If the latter case, please add a `#TODO` for future us.

Reply:
It's a hack.  But I don't plan on ever actually fixing it, per se.
I plan on making sure works when we
(1) switch to do our own Parquet files; and the (2) switch to using DM Science Pipelines-produced outputs.

I've add a note to that effect in comments next to the relevant line.